src/include/sof/audio/buffer.h

```diff
@@ -110,6 +110,7 @@ struct comp_buffer {
 	uint16_t chmap[SOF_IPC_MAX_CHANNELS];	/**< channel map - SOF_CHMAP_ */
 
 	bool hw_params_configured; /**< indicates whether hw params were set */
+	bool walking;	/**< indicates if the buffer is being walking */
```

Comment:
"walked"

Reply:
> "walked"

It does means "walking" in the implementation, not "walked".